packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -712,29 +713,29 @@ curl -v -X POST \
 | type     | The channel type. Currently the only supported type is `HTTP`.                  | channelType                             | FALSE    | FALSE  | TRUE|Must match a valid channel type.             |
-| version  | Version of the channel. The currently-supported version is "1.0.0".             | String                                  | FALSE    | FALSE  | Must match a valid version number.                |
-| config   | Properties of the communications channel used to contact your external service. | [Channel Config object](#config-object) | FALSE    | FALSE  | Validation is determined by the specific channel. |
+| version  | Version of the channel. The currently-supported version is "1.0.0".             | String                                  | FALSE    | FALSE  | Must match a valid version number                |
+| config   | Properties of the communications channel that is used to contact your external service. | [Channel Config object](#config-object) | FALSE    | FALSE  | Validation is determined by the specific channel. |
 
 
 ### Config object
 
 | Property   | Description                                                                                                | DataType                                | Required | Unique | ReadOnly | Validation                                                                                                             |
 |------------|------------------------------------------------------------------------------------------------------------|-----------------------------------------|----------|--------|----------|------------------------------------------------------------------------------------------------------------------------|
-| uri        | External service endpoint to call to execute the Inline Hook handler.                                      | String                                  | TRUE     | FALSE  | TRUE     | Must begin with `https://`. Maximum length 1024 characters. No white space allowed. The URI must be reachable by Okta. |
-| headers    | Optional list of key/value pairs for headers that should be sent with the request to the external service. | JSON object                             | FALSE    | FALSE  | FALSE    | Some reserved headers, such as `Accept`, are disallowed.                                                               |
+| uri        | External service endpoint to call to execute the Inline Hook handler                                      | String                                  | TRUE     | FALSE  | TRUE     | Must begin with `https://`. Maximum length 1024 characters. No white space allowed. The URI must be reachable by Okta. |
+| headers    | An optional list of key/value pairs for headers that should be sent with the request to the external service | JSON object                             | FALSE    | FALSE  | FALSE    | Some reserved headers, such as `Accept`, are disallowed.                                                               |
```

Comment:
An optional list of key/value pairs for headers that you should send with the request to the external service